### PR TITLE
Jetpack Cloud: Use Actual Sites for Site Count 

### PR DESCRIPTION
**Pull request by cbauerman**
#### Changes proposed in this Pull Request

* Add config option to count the number of sites available instead of using the value on the user info
* Use the new config option for Jetpack Cloud

Untitled 4

#### Testing instructions

**!!! Because this PR involves config changes you must stop and restart the build !!!**

1. Boot Jetpack Cloud and navigate to any page with any site
2. Click "Switch Site" and confirm the count matches the actual amount of sites available
3. Boot default Calypso and navigate to any page with any site
4. Click "Switch Site" and confirm the count matches production ( WordPress.com )

### CODE REVIEW

File: config/jetpack-cloud-development.json [hunk 1]
```diff
@@ -16,17 +16,18 @@
...
-		"jetpack-cloud/backups": true,
 		"jetpack-cloud/backups-restore": true,
-		"jetpack-cloud/settings": true,
-		"jetpack-cloud/scan": true,
-		"jetpack-cloud/scan-history": true,
+		"jetpack-cloud/backups": true,
 		"jetpack-cloud/on-demand-scan": true,
+		"jetpack-cloud/scan-history": true,
+		"jetpack-cloud/scan": true,
+		"jetpack-cloud/settings": true,
```
Comment (cbauerman) [inline config/jetpack-cloud-development.json:24]: alpha sort

**Comment by mattgawarecki** [comment]
I'm getting some strange behavior on my end when I test this PR, but I wonder if it's something tied to my account. Could someone with more insight help me interpret what I'm seeing?

On **WordPress.com** (production), I'm a member of 32 blogs that are split out as follows:

| Total | 32 |
| :--- | ---: |
| My Blogs | 28 |
| - Visible | 1 |
| - Hidden | 27 |
| Linked Blogs | 4 |
| - Visible | 3 |
| - Hidden | 1 |

Given that information, this is what I see on my local working copy:

**Jetpack Cloud** (this branch):
* In the sidebar site counter: **5** (with 5 little favicons, 2 being sites that are hidden)
* When I open the site switcher: a list of **3 sites** (1 that's hidden)
* In the browser's DevTools: an API response that lists **4 sites** (2 visible, 2 hidden)

**Calypso** (this branch):
* In the sidebar site counter: **4** (with 10 little favicons)
* When I open the site switcher: a list of **2 sites** (both visible)

**Comment by rcanepa** [comment]
Things work fine when I tested this branch with my account. The sites I see on JP Cloud seem to be the right ones. Calypso shows me more sites that are not relevant to JP Cloud (internal sites linked to my account).

When I open the site selector, I see the same number of sites that I see when it's closed (the sidebar component).

There is one tiny detail though, JP Cloud tells me that I have a list of 8 sites to choose from and that there are 82 more hidden. Calypso tells me that I can choose from a list of 17 sites and that there are 82 more hidden. So, I one case I seem to have 8 + 82 sites, while in the other I seem to have 17 + 82.

**Comment by mattgawarecki** [comment]
Looks good from my side! 👍 🎉 

image

**Pull request merged**